kubernetes/volume/v1alpha1/build.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	errors "github.com/openebs/maya/pkg/errors/v1alpha1"
```

Comment:
why are we using the package from maya?

Reply:
Was thinking of this too -- either reuse all OR use all from here, I suppose, for the builder context

Reply:
if we use all from maya then some of the part is not there as @amitbhatt818 mentioned earlier then we have to send PR to maya then, So I think better to use from here only